packages/hash/api/src/model/knowledge/linkEntity.model.ts

```diff
@@ -166,21 +167,65 @@ export default class extends EntityModel {
   }
 
   /**
-   * Update the link without modifying the indexes of its sibling links.
+   * Update an entity.
+   *
+   * @param params.properties - the properties object of the entity
+   * @param params.actorId - the id of the account that is updating the entity
+   */
+  async update(
+    graphApi: GraphApi,
+    params: {
+      properties: PropertyObject;
+      linkOrder: EntityLinkOrder;
```

Comment:
Nit: `linkOrder` is missing from the JSDocs

Reply:
Fixed